lib/solr_ead/behaviors.rb

```diff
@@ -118,9 +118,9 @@ def get_title(xml)
   end
 
   # Returns true or false for a component with attached <c> child nodes.
-  def component_children?(node, t = Array.new)
-    node.children.each { |n| t << n.name }
-    t.include?("c")
+  def component_children?(node)
+    node.children.each { |n| return true if n.name == 'c' }
```

Comment:
Could we use [`Enumerable#any?`](https://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-any-3F) here? 

It could make this method much more succinct